setup.py

```diff
@@ -9,7 +9,15 @@
               library_dirs=['fastFM/', 'fastFM-core/bin/'],
               include_dirs=['fastFM/', 'fastFM-core/include/',
                             'fastFM-core/externals/CXSparse/Include/',
-              numpy.get_include()])]
+                            numpy.get_include()]),
+    Extension('ffm2', ['fastFM/ffm2.pyx'],
+              #libraries=['m', 'fastfm', 'fastFM'],
+              library_dirs=['fastFM/',# 'fastFM2/build/Release/',
+                            'fastFM2/_builds/fastFM'],
```

Comment:
This path looks suspicious. Please make sure that you compile fastFM2 (manually) and then check where the lib gets saved. I would expect some `.../Release/` or `.../Debug/` location.